### ISSUE TITLE
After 15 pages the next button disappears from Reddit

### ISSUE THREAD

**Issue by nabakin** [eleweek/SearchingReddit#11]
After 15 pages of clicking the next button, the next button disappears all together.  This messes up the crawler and disables it from crawling any farther.  In order to work around this, I recommend just adding 25 to the count parameter in the url http://www.reddit.com/r/learnprogramming/?count=775 because going via the next button is sending a second parameter that restricts you from going farther than 15 pages.

**Comment by eleweek** [comment]
Thank you for reporting this problem! I noticed that the crawler crashed after some while, but didn't investigate it further.

**Comment by nabakin** [comment]
No problem!  I actually just realized that it's not always 15 pages, it fluctuates.  Sometimes it's 15, others it's 40.  It's so odd.  I also realized that my original solution won't work.  Just increasing the count parameter by 25 doesn't change the page.  You would have to figure out how to predict the next 'after' parameter in the url, not the count parameter.  Right now the crawler is kind of at the mercy of the page limit, I would absolutely love a workaround!  I never said this, but thanks for creating this and your tutorials!  They are extremely helpful and entertaining!
